sdk/tests/deqp/modules/shared/glsLifetimeTests.js

```diff
@@ -992,11 +992,11 @@ glsLifetimeTests.OutputAttachmentTest.prototype.iterate = function() {
     var element = elementType.gen();
 
     attacher.initAttachment(0, element);
+    this.m_outputAttacher.drawAttachment(element, refSurface);
     attacher.attach(element, container);
```

Comment:
Hi ken, can you explain what will the new function "detachSecondaryBindings" do for es3fLifetimeTests.BufferTfAttacher?